### ISSUE TITLE
Extra farmOS-map-chunk-*.js.LICENSE.txt files created

### ISSUE THREAD

**Issue by mstenta** [farmOS/farmOS-map#137]
Just noticed that we're creating a bunch of extra `farmOS-map-chunk-*.js.LICENSE.txt` files in the packaged releases:

```
root@d06397fedaa8:/opt/drupal# ls -al web/libraries/farmOS-map/
total 624
drwxr-xr-x 2 www-data www-data   4096 Sep 13 16:18 .
drwxr-xr-x 3 www-data www-data   4096 Sep 26 13:26 ..
-rw-r--r-- 1 www-data www-data    233 Sep 13 16:18 farmOS-map-chunk-1035a45deaad9c087345.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-1035a45deaad9c087345.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data   9064 Sep 13 16:18 farmOS-map-chunk-13e8e7684df1e30b8f47.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-13e8e7684df1e30b8f47.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data  12794 Sep 13 16:18 farmOS-map-chunk-22be7f44929bde3f5135.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-22be7f44929bde3f5135.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data    233 Sep 13 16:18 farmOS-map-chunk-39735a460ef847f90727.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-39735a460ef847f90727.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data   5699 Sep 13 16:18 farmOS-map-chunk-54a6c2fa30f9ee04a341.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-54a6c2fa30f9ee04a341.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data    835 Sep 13 16:18 farmOS-map-chunk-563e7276f62b25852291.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-563e7276f62b25852291.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data   1796 Sep 13 16:18 farmOS-map-chunk-647bd575b50bb76e6f50.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-647bd575b50bb76e6f50.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data  23749 Sep 13 16:18 farmOS-map-chunk-72ba8dece6e00934eef6.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-72ba8dece6e00934eef6.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data    233 Sep 13 16:18 farmOS-map-chunk-a0608a8045d2ff14b759.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-a0608a8045d2ff14b759.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data   1174 Sep 13 16:18 farmOS-map-chunk-a07f228d06bcfd7a3168.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-a07f228d06bcfd7a3168.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data   8381 Sep 13 16:18 farmOS-map-chunk-a817ade2fb1f26951406.css
-rw-r--r-- 1 www-data www-data   1459 Sep 13 16:18 farmOS-map-chunk-bf740d1a94ebc0d2ae34.css
-rw-r--r-- 1 www-data www-data    163 Sep 13 16:18 farmOS-map-chunk-c6cc00ebd9b98bc29a90.css
-rw-r--r-- 1 www-data www-data  22035 Sep 13 16:18 farmOS-map-chunk-e19c1353e1d0b55e0be2.js
-rw-r--r-- 1 www-data www-data     24 Sep 13 16:18 farmOS-map-chunk-e19c1353e1d0b55e0be2.js.LICENSE.txt
-rw-r--r-- 1 www-data www-data    548 Sep 13 16:18 farmOS-map-chunk-e8a93b5be84cf9b8168d.css
-rw-r--r-- 1 www-data www-data    538 Sep 13 16:18 farmOS-map-chunk-eeb18cd7843f780ff96a.css
-rw-r--r-- 1 www-data www-data    315 Sep 13 16:18 farmOS-map-chunk-f4eae938edcdeae88d3b.css
-rw-r--r-- 1 www-data www-data  20106 Sep 13 16:18 farmOS-map.css
-rw-r--r-- 1 www-data www-data 416389 Sep 13 16:18 farmOS-map.js
-rw-r--r-- 1 www-data www-data    214 Sep 13 16:18 farmOS-map.js.LICENSE.txt
```

The contents of each is simply:

```
/*! farmOS-map 2.0.2 */
```

**Comment by mstenta** [comment]
I assume `BannerPlugin` is the cause of this: https://github.com/farmOS/farmOS-map/blob/8babd7d7bc46156355ac8bd30043e554bd91df64/webpack.config.js#L34

We only needed `BannerPlugin` to put the version in `farmOS-map.js` for farmOS 7.x-1.x, so that the Drupal 7 Libraries module could check the version: https://github.com/farmOS/farmOS/blob/722339943b4ab30587eb0795249f0e1d2867f838/modules/farm/farm_map/farm_map.module#L82

We don't need this in farmOS 2.x. So perhaps we should just remove `BannerPlugin`.

**Comment by mstenta** [comment]
Pull request for consideration: https://github.com/farmOS/farmOS-map/pull/138

**Comment by mstenta** [comment]
I've closed my PR (#138) in favor of @symbioquine's #139.

**Comment by mstenta** [comment]
Fixed by #139 - thanks @symbioquine !

**Comment by symbioquine** [comment]
Seems like we can probably close this right @mstenta?
